uart_raw.h

```diff
@@ -15,6 +15,8 @@ void uart_init();
 void uart_putch(unsigned char c);
 /* Transmits a character string via UART1. */
 void uart_putstr(const char* str);
+/* Transmits n characters from a string via UART1. */
+void uart_putnstr(unsigned int n, const char* str);
```

Comment:
Please be consistent with C standard library function naming and argument passing order, if possible.

int uart_putchar(int c);
int uart_puts(const char *s);
int uart_write(const void *s, size_t n);